tools/devtool

```diff
@@ -486,7 +486,7 @@ cmd_build() {
 }
 
 function cmd_make_release {
-    cmd_test -- --reruns 2 || die "Tests failed!"
+    cmd_test -- --reruns 2 -n 8 || die "Tests failed!"
```

Comment:
I really want to do this, but I think we need to be careful .Some of the tests cannot be run in parallel. Like the build ones. The performance ones may also fail since they assume they have the whole host to work with.

We could separate it in two runs.

```
    OPTS="--reruns=7"
    cmd_test -- $OPTS --json-report-file=test-report.json -n8 --dist=worksteal integration_tests/{functional,security,style} || die "Tests failed!"
    cmd_test -- $OPTS --json-report-file=test-report-perf.json integration_tests/{performance,build} || die "Tests failed!"
```

We end up with 2 files, so we would need to adapt the release script to account for the other file.

Reply:
Ahhh, I did not realize this also ran the build and performance test. Suffer more I shall then.

Reply:
Also if we do this we may be able to run the release sanity check without faking the testrun: https://github.com/firecracker-microvm/firecracker/blob/main/.buildkite/pipeline_pr.py#L52